stories/visually-hidden.stories.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { storiesOf } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import VisuallyHidden from '../assets/js/components/VisuallyHidden';
+
+storiesOf( 'Global', module )
+	.add( 'Visually Hidden', () => (
+		<div>
+			VisuallyHidden:
+			<div style={ { padding: '10px', background: '#e3e3e3', display: 'inline-block', verticalAlign: 'text-bottom' } }>
+				<VisuallyHidden style={ { background: '#fff' } }>
+					{ __( 'Child Content1', 'google-site-kit' ) }
+				</VisuallyHidden>
+			</div>
+
+			<br />
+			<br />
```

Comment:
Use `margin` instead of these `<br/>` tags.

Reply:
@eugene-manuilov While running ` npm run test:visualtest`, I am getting
```
ENGINE ERROR: Protocol error (Page.navigate): Cannot navigate to invalid URL
```
and sometimes 
```
Puppeteer encountered an error while running scenario "Global/Admin Bar"
TimeoutError: Navigation Timeout Exceeded: 60000ms exceeded
```

Any clues how to solve this? Also is there a special way to regenerate reference images for backstop?

Reply:
@kostyalmm, do you see which URL it tries to navigate to?

> Puppeteer encountered an error while running scenario "Global/Admin Bar"
> TimeoutError: Navigation Timeout Exceeded: 60000ms exceeded

Seems like something wrong with that story setup because the admin bar component shows preview components instead of actual statistic... Hard to say exactly why it happens, I have never configured it. @aaemnnosttv do you have any idea why admin bar has started causing issues?

Reply:
it is not only admin bar, all components are showing this error

Reply:
@kostyalmm perhaps you may have deleted a reference file accidentally? Does that file exist for you locally?

Looking at the VRT that's failing in CI, only the 3 screens are failing for the new `Global/Visually Hidden` scenario due to a change in padding or margin.
![image](https://user-images.githubusercontent.com/1621608/98259079-5233b380-1f8a-11eb-8126-90003f3aa0ea.png)

I see the same when running it locally as well and `npm run test:visualtest` runs fine for me on your branch so I'm not sure what the problem may be.

Reply:
I ended up redoing the whole setup and it worked. It was not able to generate the test file.